packages/core/core-flows/src/cart/workflows/create-payment-collection-for-cart.ts

```diff
@@ -130,5 +132,7 @@ export const createPaymentCollectionForCartWorkflow = createWorkflow(
     createRemoteLinkStep(cartPaymentLink).config({
       name: "cart-payment-collection-link",
     })
+
+    return new WorkflowResponse(created)
```

Comment:
nit: this flow only creates one payment collection, we could return just `created[0]`